benchmarks/server-router.js

```diff
@@ -30,47 +30,47 @@ routes.forEach(route => {
 
 time = now()
 for (i = 0; i < operations; i++) {
-  router.match({ method: 'GET', url: '/user' })
+  router._router.match('GET/user')
```

Comment:
Are you sure this is correct?
`server-router` internally uses `wayfarer` which is not an http router, that's why is not present in the benchmarks.

Reply:
As mentioned in the PR comment, I actually think `wayfarer` qualifies as an HTTP router. It just needs a little bit of special care with HTTP verbs, which `server-router` [abstracts away in the handling mode](https://github.com/yoshuawuyts/server-router/blob/811aaaaaa93a35c039a21b637fc02ef6d6fcb35a/index.js#L37-L41).